docs/LocalDevelopmentGuide.md

```diff
@@ -76,82 +89,79 @@ We send our logs to CloudWatch for easier storage and querying.
 Finally, copy the provided access key ID and secret access key after creating
 the new user.
 
-Note: if you are in the `brussel-sprouts` Github team, you should already have
+Note: if you are in the `brussel-sprouts` GitHub team, you should already have
 AWS credentials. Just ask.
 
+### Using Local AWS
+
 Alternatively, just set up [DynamoDB locally][localdynamodb] (the Docker-based
 setup is probably the easiest) and set `AWS_LOCAL=True`.
 
-## 5: Set Up Config
-
-Our repo already contains `sample-env`, the main environmental configuration
-file for the entire app, as well as the `credentials/` directory, where you
-will put credential files like the Github app private key. The file is split
-into section. There is a general section (which should be the top bit), a
-section on everything slack related, a section on Github and Github apps,
-and a section on AWS. [Please read the section about the configuration
-system.][config]
+CloudWatch integration is not currently supported in this manner.
 
-### 5.1: Set up Github App and organization
+### 5: Set up a GitHub App and organization
 
-Register Rocket 2 as a Github App under an appropriate testing organization
-(our team has one of these set up already). Make sure to install the Github App
-to the organization in addition to registering it.
+Create a Rocket 2 Github under an appropriate testing organization. Make sure
+to install the GitHub App to the organization in addition to registering it.
+All this can be done from a GitHub organization's Settings > GitHub Apps page.
 
-Under "Private keys", click "Generate a new private key". This will generate
-and allow you to download a new secret key for Rocket 2. Save this to the
-`credentials/` directory as `github_signing_key.pem` - it should already be in
-the PEM file format, bracketed by:
+In the GitHub app's settings, go to "Private keys" and click "Generate a new
+private key". This will generate and allow you to download a new secret key for
+Rocket 2. Save this to the `credentials/` directory as `gh_signing_key.pem` -
+it should already be in the PEM file format, bracketed by:
```

Comment:
This seems out of date. The current way to do things would be to shove the GitHub Key as an environmental variable in your `.env`.

Reply:
This is still right, the first step is to download it